1.Practicals_C/Miscc/profit or loss.c

```diff
@@ -0,0 +1,16 @@
+#include <stdio.h>
+int main()
+{
+    float cp, sp, p, l;
+    printf("\nEnter cost price and selling price:");
+    scanf("%f %f", &cp, &sp);
+    p = sp - cp;
```

Comment:
This logic can be improved

Reply:
You can have something like,
```
if (p>0)
        printf("Profit: %f \n", p);
else if (p< 0)
        printf("Loss: %f \n", -p);
else
 printf("No profit, No loss ")
```

This way, computing l wouldn't be required. And, the logic would be computationally efficient.